src/ReactiveUI.Tests/Platforms/xaml/ActivationForViewFetcherTest.cs

```diff
@@ -95,14 +98,14 @@ public void IsHitTestVisibleDeactivatesFrameworkElement()
             new[] { true, false }.AssertAreEqual(activated);
         }
 
-        [Fact]
+        [WpfFact]
         public void FrameworkElementIsActivatedAndDeactivatedWithHitTest()
         {
             var uc = new TestUserControl();
             var activation = new ActivationForViewFetcher();
 
             var obs = activation.GetActivationForView(uc);
-            var activated = obs.CreateCollection();
+            var activated = obs.CreateCollection(scheduler: ImmediateScheduler.Instance);
```

Comment:
I get the feeling this shouldn't be necessary, but it looks like the way RxApp works, sometimes MainThreadScheduler might not be set as ImmediateScheduler in unit tests (we set _MainThreadScheduler but _UnitTestScheduler wins in general).